content/docs/start/data-and-model-versioning.md

```diff
@@ -25,16 +25,16 @@ To start tracking a file or directory, use `dvc add`:
 
 ### ⚙️ Expand to get an example dataset.
 
-Having initialized a project in the previous section, get the data file we will
-be using later like this:
+Having initialized a project in the previous section, get the data file which we
```

Comment:
Hmmm. Also not sure about "which" here ("which we will be using" isn't super readable). But the current text isn't great either. Maybe rewrite the whole paragraph?

Reply:
reworded